lib/Basic/Version.cpp

```diff
@@ -43,6 +43,12 @@
   SWIFT_MAKE_VERSION_STRING(SWIFT_VERSION_MAJOR, SWIFT_VERSION_MINOR)
 #endif
 
+// MSVC doesn't support __has_include
```

Comment:
Ah I understand now. I've modified `lib/Basic/CMakeLists.txt` file to generate an empty file if these files are not found. I've made it a seperate commit because its not necessarily Windows/MSVC specific.